tests/test_pgbouncer.py

```diff
@@ -0,0 +1,67 @@
+import unittest
+import subprocess
+from tests.common import load_check
+
+from nose.plugins.attrib import attr
+
+import time
+from pprint import pprint
+
+@attr(requires='pgbouncer')
+class TestPgbouncer(unittest.TestCase):
+
+    def test_checks(self):
+
+        config = {
+            'instances': [
+                {
+                'host': 'localhost',
+                'port': 15433,
+                'username': 'datadog',
+                'password': 'datadog'
+                }
+            ]
+        }
+        agentConfig = {
+            'version': '0.1',
+            'api_key': 'toto'
+        }
+
+        self.check = load_check('pgbouncer', config, agentConfig)
+        self.check.run()
+        metrics = self.check.get_metrics()
+        self.assertTrue(len([m for m in metrics if m[0] == u'pgbouncer.pools.cl_active'])           >= 1, pprint(metrics))
+        self.assertTrue(len([m for m in metrics if m[0] == u'pgbouncer.pools.cl_waiting'])          >= 1, pprint(metrics))
+        self.assertTrue(len([m for m in metrics if m[0] == u'pgbouncer.pools.sv_active'])           >= 1, pprint(metrics))
+        self.assertTrue(len([m for m in metrics if m[0] == u'pgbouncer.pools.sv_idle'])             >= 1, pprint(metrics))
+        self.assertTrue(len([m for m in metrics if m[0] == u'pgbouncer.pools.sv_used'])             >= 1, pprint(metrics))
+        self.assertTrue(len([m for m in metrics if m[0] == u'pgbouncer.pools.sv_tested'])           >= 1, pprint(metrics))
+        self.assertTrue(len([m for m in metrics if m[0] == u'pgbouncer.pools.sv_login'])            >= 1, pprint(metrics))
+        self.assertTrue(len([m for m in metrics if m[0] == u'pgbouncer.pools.maxwait'])             >= 1, pprint(metrics))
+
+        self.assertTrue(len([m for m in metrics if m[0] == u'pgbouncer.stats.total_query_time'])    >= 1, pprint(metrics))
+        self.assertTrue(len([m for m in metrics if m[0] == u'pgbouncer.stats.avg_req'])             >= 1, pprint(metrics))
+        self.assertTrue(len([m for m in metrics if m[0] == u'pgbouncer.stats.avg_recv'])            >= 1, pprint(metrics))
+        self.assertTrue(len([m for m in metrics if m[0] == u'pgbouncer.stats.avg_sent'])            >= 1, pprint(metrics))
+        self.assertTrue(len([m for m in metrics if m[0] == u'pgbouncer.stats.avg_query'])           >= 1, pprint(metrics))
+        # Rate metrics, need 2 collection rounds
+        subprocess.call(['embedded/pgbouncer/pgbouncer_stimulus.sh'])
```

Comment:
One solution (see the comment on that script) would be to call in psycopg2 and make a request to the DB with it.